src/pointarray.js

```diff
@@ -87,8 +87,8 @@ SVG.extend(SVG.PointArray, {
 
     // recalculate position of all points according to new size
     for (i = this.value.length - 1; i >= 0; i--) {
-      this.value[i][0] = ((this.value[i][0] - box.x) * width)  / box.width  + box.x
-      this.value[i][1] = ((this.value[i][1] - box.y) * height) / box.height + box.y
+      if(box.width) this.value[i][0] = ((this.value[i][0] - box.x) * width)  / box.width  + box.x
```

Comment:
So `.size()` is a noop (no operation) if the shape has no width or height? Should be written in the documentation before we get a `.size()` doesn't work issue filed.